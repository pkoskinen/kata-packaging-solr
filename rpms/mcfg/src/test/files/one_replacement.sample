foo = bar
mail = %%EMAIL%%
yes = no
