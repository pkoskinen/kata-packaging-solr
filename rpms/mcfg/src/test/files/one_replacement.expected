foo = bar
mail = admin@here.org
yes = no
